src/main/java/com/khutircraftubackend/config/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,38 @@
+package com.khutircraftubackend.config;
+
+import com.khutircraftubackend.security.JwtUtils;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+@Component
+@RequiredArgsConstructor
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+    private final JwtUtils jwtUtils;
+    private final UserDetailsService userDetailsService;
+    @Override
```

Comment:
really minor thing, but still, better readable if added extra empty line in between lines 22 and 23.
IntelliJ might display additional info, like "usages" that make you feel like there is an extra line, 
but for those who uses other IDEs and don't have the same feature, all code lines appear glued back to back with each other.

Reply:
Ok, I'll add an empty line

Reply:
not critical though :)